lib/segment/src/common/operation_time_statistics.rs

```diff
@@ -373,6 +374,15 @@ fn merge_histograms(
     total_a: usize,
     total_b: usize,
 ) -> Vec<(f32, usize)> {
+    // TODO: drop is_sorted crate and use Iterator::is_sorted once it's stable
```

Comment:
I'm not sure if this will be stabilized anytime soon, it has been open for more than 5 years already :(
https://github.com/rust-lang/rust/issues/53485

Reply:
It's more like of a reminder for someone who will touch this part of code in the future.